styles/style.css

```diff
@@ -0,0 +1,111 @@
+
+.frame{
+	width: 460px;
+	height: 488px;
+	background: #2F4F4F;
+	border-radius: 15px;
+	margin: auto;
+}
+
+
+
+
+
+.profile{
+	float: left;
+	width: 220px;
+	background:#00BFFF;
+	margin: 17px 0 0 17px;
+	height: 452px;
+	border-radius: 15px;
+	text-align: center;}
+
+.profile p{
+	margin: 8px 0 0 0;
+	font-size: 20px;}
+	.profile img{
+		width: 110px;
+		border-radius: 50%;
+		margin: 37px 0 20px 0;}
+
+	img:hover{
+		transform: rotate(360deg);
+		transition: all 1s ease-in-out;}
+
+
+
+
+.button{
+	margin: 28px 0 0 0;}
+	.button1{
```

Comment:
all buttons can have the same styles, you don't need to add width, and can add the same margin
so create one common class for the button, and remove all duplicates